packages/devtools_app/lib/src/screens/provider/instance_viewer/fake_freezed_annotation.dart

```diff
@@ -7,20 +7,10 @@
 // We could instead remove the annotations, but that would make the process of
 // updating the generated files tedious.
 
-const nullable = Object();
```

Comment:
Revert changes to this file. It is generated.

Reply:
Ah, I see, now it fails coz of that. Will update tomorrow.

Reply:
I've reverted this change, could you please restart the pipelines?